core/storage/story/gae_models.py

```diff
@@ -0,0 +1,199 @@
+# Copyright 2018 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Models for storing the story data models."""
+
+from core.platform import models
+
+from google.appengine.ext import ndb  # pylint: disable=relative-import
+
+(base_models, user_models,) = models.Registry.import_models([
+    models.NAMES.base_model, models.NAMES.user])
+
+
+class StorySnapshotMetadataModel(base_models.BaseSnapshotMetadataModel):
+    """Storage model for the metadata for a story snapshot."""
+    pass
+
+
+class StorySnapshotContentModel(base_models.BaseSnapshotContentModel):
+    """Storage model for the content of a story snapshot."""
+    pass
+
+
+class StoryModel(base_models.VersionedModel):
+    """Model for storing stories.
+
+    This class should only be imported by the story domain file, the
+    story services file, and the story model test file.
+    """
+    SNAPSHOT_METADATA_CLASS = StorySnapshotMetadataModel
+    SNAPSHOT_CONTENT_CLASS = StorySnapshotContentModel
+    ALLOW_REVERT = True
```

Comment:
Not sure this needs to allow reverts either.

Reply:
Done.